src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -213,8 +212,20 @@ protected final String buildMethodName() {
     }
 
     protected final AnnotationSpec generatedAnnotation() {
+        Class<?> generatedAnnotation;
+        try {
+            // available since 9
+            generatedAnnotation = Class.forName("javax.annotation.processing.Generated");
+        } catch (ClassNotFoundException e) {
+            try {
+                generatedAnnotation = Class.forName("javax.annotation.Generated");
+            } catch (ClassNotFoundException ex) {
+                // Shouldn't be possible.
+                throw new IllegalStateException(ex);
+            }
```

Comment:
Instead of this dead code, can you instead change this method to have `throws Exception` (Its caller already has it, so it shouldn't cause any problems), and remove this entire `catch` statement?

Reply:
Done